package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "react-js-mentoring",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --open --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js",
```

Comment:
According to task acceptance criteria we should use ENV variables to distinguish prod and dev versions.